SU2_CFD/src/numerics/turbulent/turb_sources.cpp

```diff
@@ -761,9 +761,13 @@ CSourcePieceWise_TurbSST::CSourcePieceWise_TurbSST(unsigned short val_nDim,
 
   incompressible = (config->GetKind_Regime() == INCOMPRESSIBLE);
   sustaining_terms = (config->GetKind_Turb_Model() == SST_SUST);
+  implicit = (config->GetKind_TimeIntScheme_Flow() == EULER_IMPLICIT);
```

Comment:
Correct me if Im wrong here, but aren't the turbulent equations always implicit?

Also, I believe this would have to be GetKind_TimeIntScheme_Turb?

Reply:
It is ok to always compute the Jacobian when they are this simple, and let the solver decide if it wants to use the result or not. It can potentially solve a lot of head ache.

Reply:
Alright I see... thanks. Well spotted copy paste bug!